app/models/workers_scaling/experiment_resources_interface.rb

```diff
@@ -6,16 +6,17 @@ module WorkersScaling
   ##
   # Experiment interface to schedule and maintain computational resources
   class ExperimentResourcesInterface
+    MAXIMUM_NUMBER_OF_FAILED_WORKERS = 3
 
     ##
     # Params:
     # * experiment
     # * user_id
     # * allowed_infrastructures
-    #     Variable storing user-defined workers limits for each infrastructure
+    #     Variable storing user-defined workers limits for each infrastructure configuration
     #     Only infrastructures specified here can be used by experiment.
-    #     Format: [{infrastructure: <infrastructure>, limit: <limit>}, ...]
-    #     <infrastructure> - hash with infrastructure info
+    #     Format: [{infrastructure: <infrastructure_configuration>, limit: <limit>}, ...]
```

Comment:
Jak mówiłem kiedyś, żeby zmienić nazwę subinfrastructure, bo już sam nie pamiętam? Właściwie, formalnie to jest resource_configuration (konfiguracja zasobu, który otrzymamy od infrastruktury), bo nie konfigurujemy infrastruktury tylko specyfikujemy jaki zasób chcemy otrzymać.

Jeśli ta zmiana sprowadzałaby się do znajdź/zamień to zmienić. Jeśli coś się miałoby przy tym posypać to zostawić.